index.bs

```diff
@@ -2663,6 +2667,8 @@ during credential generation.
     ::  This value indicates that the [=[RP]=] wants to receive the [=attestation statement=] as generated by the
         [=authenticator=].
 
+    :   <dfn>enterprise</dfn>
+    ::  This value indicates that the [=[RP]=] wants to receive an [=attestation statement=] that may include uniquely identifying information. This is intended for controlled deployments within an enterprise where the organization wishes to tie registrations to specific authenticators. User agents MUST NOT provide such an attestation unless user agent or authenticator configuration permits it for the requested [=RP ID=]. If not permitted, the user agent SHOULD treat this value the same as `direct`.
```

Comment:
```suggestion
    ::  This value indicates that the [=[RP]=] wants to receive an [=attestation statement=] that may include uniquely identifying information. This is intended for controlled deployments within an enterprise where the organization wishes to tie registrations to specific authenticators. User agents MUST NOT provide such an attestation unless user agent or authenticator configuration permits it for the requested [=RP ID=]. If not permitted, the user agent SHOULD treat this value the same as `direct`. 
    
    If permitted, the user agent SHOULD signal to the authenticator (at [=#CreateCred-InvokeAuthnrMakeCredauthenticator|invocation time=]) that enterprise attestation is requested, and convey the resulting [=AAGUID=] and [=attestation statement=], unaltered, to the [=[RP]=]. 
```

Reply:
Sure this shouldn't be a new parameter of `authenticatorMakeCredential`?

Reply:
Maybe, but I don't think so? For example, what would a call with enterprise + none-attestation mean? I.e. I don't think the cross-product is meaning in several cases thus I'm casting this as an alternative.